---
title: "Research"
author: "NJ Grunwald"
date: 2018-05-17T21:13:14-05:00
categories: ["R"]
weight: 30
bibliography: [refs.bib]

---



<p>Welcome to the Grunwald lab at the USDA ARS, Horticultural Crops Research Unit. Our lab focuses on developing approaches for studying and characterizing emerging plant pathogens. Specifically we focus on:</p>
<div id="emergence-of-plant-pathogens" class="section level2">
<h2>Emergence of plant pathogens</h2>
<p>Our work focuses on characterizing several emerging plant pathogens. One focus of our work is the sudden oak death pathogen <em>Phytophthora ramorum</em>:</p>
<div class="float">
<img src="static/img/ramorum%20emergence.png" alt="Emergence of Phytophthora ramorum" />
<div class="figcaption">Emergence of <em>Phytophthora ramorum</em></div>
</div>
<p>The recently emerged plant pathogen <em>Phytophthora ramorum</em> is responsible for causing the sudden oak death epidemic. We have documented the emergence of <em>P. ramorum</em> based on evolutionary and population genetic analyses <strong><span class="citation">(<a href="#ref-Grunwald2012emergence">Grünwald et al. 2012</a>; <a href="#ref-Goss2009PLospathogens">Goss et al. 2009</a> )</span></strong>. Currently infection by <em>P. ramorum</em> occurs only in Europe and North America and three clonal lineages are distinguished: EU1, NA1 and NA2 <strong><span class="citation">(<a href="#ref-Grunwald2008ramorum">Grünwald, Goss, and Press 2008</a>)</span></strong>. Ancient divergence of these lineages supports a scenario in which <em>P. ramorum</em> originated from reproductively isolated populations and underwent at least four global migration events <strong><span class="citation">(<a href="#ref-goss2009Ancient">Goss, Carbone, and Grünwald 2009</a>)</span></strong>. We also work on other pathogens <strong><span class="citation">(<a href="#ref-Davise01280_18">Davis et al. 2018</a>)</span></strong>. This recent work sheds new light on mechanisms of emergence of exotic pathogens and provides crucial insights into migration pathways.</p>
</div>
<div id="development-of-computational-tools" class="section level2">
<h2>Development of computational tools</h2>
<div class="float">
<img src="/img/journal.pcbi.1005404.g005.PNG" alt="Heat-trees produced with our metacoder R package" />
<div class="figcaption">Heat-trees produced with our <code>metacoder</code> R package</div>
</div>
<p>Our lab has developed several computational tools in R including <code>poppr</code>, <code>popprxl</code>, <code>vcfR</code>, <code>metacoder</code> , <code>taxa</code>, and <code>effectr</code> <strong><span class="citation">(<a href="#ref-Foster_2017">Foster, Sharpton, and Grünwald 2017</a>; <a href="#ref-Knaus_2018inferring">Brian J. Knaus and Grünwald 2018</a>; <a href="#ref-Grunwald2016_vcfr">B. J. Knaus and Grünwald 2016</a>; <a href="#ref-Kamvar_2014">Zhian N. Kamvar, Tabima, and Grünwald 2014</a>; <a href="#ref-Grunwald2015Novel">Z. N. Kamvar, Brooks, and Grünwald 2015</a>; <a href="#ref-tabima2018effectr">Tabima and Grunwald 2018</a>; <a href="#ref-Foster_2018">Foster, Chamberlain, and Grünwald 2018</a>)</span></strong>. These tools are useful for characterizing emergence of plant pathogens using approaches including population genetics, genomics, and community ecology of microbiomes. These tools are provided with extensive educational resources. We have also used these tools for several websites we maintain.</p>
</div>
<div id="select-references" class="section level1 unnumbered">
<h1>Select references</h1>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-Davise01280_18" class="csl-entry">
Davis, EW, JF Tabima, AJ Weisberg, LD Lopes, MS Wiseman, MS Wiseman, T Pupko, et al. 2018. <span>“Evolution of the u.s. Biological Select Agent Rathayibacter Toxicus.”</span> <em>mBio</em> 9 (4): e01280–18. <a href="https://doi.org/10.1128/mBio.01280-18">https://doi.org/10.1128/mBio.01280-18</a>.
</div>
<div id="ref-Foster_2018" class="csl-entry">
Foster, Zachary S. L., Scott Chamberlain, and Niklaus J. Grünwald. 2018. <span>“Taxa: An r Package Implementing Data Standards and Methods for Taxonomic Data.”</span> <em>F1000Research</em> 7 (March): 272. <a href="https://doi.org/10.12688/f1000research.14013.1">https://doi.org/10.12688/f1000research.14013.1</a>.
</div>
<div id="ref-Foster_2017" class="csl-entry">
Foster, Zachary S. L., Thomas J. Sharpton, and Niklaus J. Grünwald. 2017. <span>“Metacoder: An r Package for Visualization and Manipulation of Community Taxonomic Diversity Data.”</span> Edited by Timothee Poisot. <em>PLoS Computational Biology</em> 13 (2): e1005404. <a href="https://doi.org/10.1371/journal.pcbi.1005404">https://doi.org/10.1371/journal.pcbi.1005404</a>.
</div>
<div id="ref-goss2009Ancient" class="csl-entry">
Goss, E. M., I. Carbone, and N. J. Grünwald. 2009. <span>“Ancient Isolation and Independent Evolution of the Three Clonal Lineages of the Exotic Sudden Oak Death Pathogen Phytophthora Ramorum.”</span> <em>Molecular Ecology</em> 18 (6): 1161–74. <a href="https://doi.org/10.1111/j.1365-294X.2009.04089.x">https://doi.org/10.1111/j.1365-294X.2009.04089.x</a>.
</div>
<div id="ref-Goss2009PLospathogens" class="csl-entry">
Goss, E. M., M. Larsen, G. A. Chastagner, D. R. Givens, and N. J. Grünwald. 2009. <span>“Population Genetic Analysis Infers Migration Pathways of Phytophthora Ramorum in US Nurseries.”</span> <em>PLoS Pathogens</em> 5 (9). <a href="https://doi.org/10.1371/journal.ppat.1000583">https://doi.org/10.1371/journal.ppat.1000583</a>.
</div>
<div id="ref-Grunwald2012emergence" class="csl-entry">
Grünwald, N. J., M. Garbelotto, E. M. Goss, K. Heungens, and S. Prospero. 2012. <span>“Emergence of the Sudden Oak Death Pathogen Phytophthora Ramorum.”</span> <em>Trends in Microbiology</em> 20 (3): 131–38. <a href="https://doi.org/10.1016/j.tim.2011.12.006">https://doi.org/10.1016/j.tim.2011.12.006</a>.
</div>
<div id="ref-Grunwald2008ramorum" class="csl-entry">
Grünwald, N. J., E. M. Goss, and C. M. Press. 2008. <span>“Phytophthora Ramorum: A Pathogen with a Remarkably Wide Host Range Causing Sudden Oak Death on Oaks and Ramorum Blight on Woody Ornamentals.”</span> <em>Molecular Plant Pathology</em> 9 (6): 729–40. <a href="https://doi.org/10.1111/j.1364-3703.2008.00500.x">https://doi.org/10.1111/j.1364-3703.2008.00500.x</a>.
</div>
<div id="ref-Grunwald2015Novel" class="csl-entry">
Kamvar, Z. N., J. C. Brooks, and N. J. Grünwald. 2015. <span>“Novel r Tools for Analysis of Genome-Wide Population Genetic Data with Emphasis on Clonality.”</span> <em>Frontiers in Genetics</em> 6 (JUN). <a href="https://doi.org/10.3389/fgene.2015.00208">https://doi.org/10.3389/fgene.2015.00208</a>.
</div>
<div id="ref-Kamvar_2014" class="csl-entry">
Kamvar, Zhian N., Javier F. Tabima, and Niklaus J. Grünwald. 2014. <span>“Poppr: An r Package for Genetic Analysis of Populations with Clonal, Partially Clonal, and/or Sexual Reproduction.”</span> <em>PeerJ</em> 2 (March): e281. <a href="https://doi.org/10.7717/peerj.281">https://doi.org/10.7717/peerj.281</a>.
</div>
<div id="ref-Grunwald2016_vcfr" class="csl-entry">
Knaus, B. J., and N. J. Grünwald. 2016. <span>“Vcfr: A Package to Manipulate and Visualize Variant Call Format Data in r.”</span> <em>Molecular Ecology Resources</em>. <a href="https://doi.org/10.1111/1755-0998.12549">https://doi.org/10.1111/1755-0998.12549</a>.
</div>
<div id="ref-Knaus_2018inferring" class="csl-entry">
Knaus, Brian J., and Niklaus J. Grünwald. 2018. <span>“Inferring Variation in Copy Number Using High Throughput Sequencing Data in r.”</span> <em>Frontiers in Genetics</em> 9 (April). <a href="https://doi.org/10.3389/fgene.2018.00123">https://doi.org/10.3389/fgene.2018.00123</a>.
</div>
<div id="ref-tabima2018effectr" class="csl-entry">
Tabima, Javier F, and Niklaus J Grunwald. 2018. <span>“effectR: An Expandable r Package to Predict Candidate Effectors.”</span> <em>BioRxiv</em>, 398404. <a href="https://doi.org/10.1094/MPMI-10-18-0279-TA">https://doi.org/10.1094/MPMI-10-18-0279-TA</a>.
</div>
</div>
</div>
