---
title: "Research"
author: "NJ Grunwald"
date: 2018-05-17T21:13:14-05:00
categories: ["R"]
weight: 30
bibliography: [refs.bib]

---

<link href="/rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="/rmarkdown-libs/anchor-sections/anchor-sections.js"></script>


<p>Welcome to the Grunwald lab at the USDA ARS, Horticultural Crops Research Unit. Our lab focuses on developing approaches for studying and characterizing emerging plant pathogens. Specifically we focus on:</p>
<div id="emergence-of-plant-pathogens" class="section level2">
<h2>Emergence of plant pathogens</h2>
<p>Our work focuses on characterizing several emerging plant pathogens. One focus of our work is the sudden oak death pathogen <em>Phytophthora ramorum</em>:</p>
<div class="figure">
<img src="/img/ramorum%20emergence.png" alt="" />
<p class="caption">Emergence of <em>Phytophthora ramorum</em></p>
</div>
<p>The recently emerged plant pathogen <em>Phytophthora ramorum</em> is responsible for causing the sudden oak death epidemic. We have documented the emergence of <em>P. ramorum</em> based on evolutionary and population genetic analyses <strong><span class="citation">(Grünwald et al. 2012; Goss et al. 2009 )</span></strong>. Currently infection by <em>P. ramorum</em> occurs only in Europe and North America and three clonal lineages are distinguished: EU1, NA1 and NA2 <strong><span class="citation">(Grünwald, Goss, and Press 2008)</span></strong>. Ancient divergence of these lineages supports a scenario in which <em>P. ramorum</em> originated from reproductively isolated populations and underwent at least four global migration events <strong><span class="citation">(Goss, Carbone, and Grünwald 2009)</span></strong>. We also work on other pathogens <strong><span class="citation">(Davis et al. 2018)</span></strong>. This recent work sheds new light on mechanisms of emergence of exotic pathogens and provides crucial insights into migration pathways.</p>
</div>
<div id="development-of-computational-tools" class="section level2">
<h2>Development of computational tools</h2>
<div class="figure">
<img src="/img/journal.pcbi.1005404.g005.PNG" alt="" />
<p class="caption">Heat-trees produced with our <code>metacoder</code> R package</p>
</div>
<p>Our lab has developed several computational tools in R including <code>poppr</code>, <code>popprxl</code>, <code>vcfR</code>, <code>metacoder</code> , <code>taxa</code>, and <code>effectr</code> <strong><span class="citation">(Foster, Sharpton, and Grünwald 2017; Knaus and Grünwald 2018, 2016; Kamvar, Tabima, and Grünwald 2014; Kamvar, Brooks, and Grünwald 2015; Tabima and Grunwald 2018; Foster, Chamberlain, and Grünwald 2018)</span></strong>. These tools are useful for characterizing emergence of plant pathogens using approaches including population genetics, genomics, and community ecology of microbiomes . These tools are provided with extensive educational resources. We have also used these tools for several websites we maintain.</p>
</div>
<div id="select-references" class="section level1 unnumbered">
<h1>Select references</h1>
<div id="refs" class="references">
<div id="ref-Davise01280_18">
<p>Davis, EW, JF Tabima, AJ Weisberg, LD Lopes, MS Wiseman, MS Wiseman, T Pupko, et al. 2018. “Evolution of the U.s. Biological Select Agent Rathayibacter Toxicus.” <em>mBio</em> 9 (4): e01280–18. <a href="https://doi.org/10.1128/mBio.01280-18">https://doi.org/10.1128/mBio.01280-18</a>.</p>
</div>
<div id="ref-Foster_2018">
<p>Foster, Zachary S. L., Scott Chamberlain, and Niklaus J. Grünwald. 2018. “Taxa: An R Package Implementing Data Standards and Methods for Taxonomic Data.” <em>F1000Research</em> 7 (March): 272. <a href="https://doi.org/10.12688/f1000research.14013.1">https://doi.org/10.12688/f1000research.14013.1</a>.</p>
</div>
<div id="ref-Foster_2017">
<p>Foster, Zachary S. L., Thomas J. Sharpton, and Niklaus J. Grünwald. 2017. “Metacoder: An R Package for Visualization and Manipulation of Community Taxonomic Diversity Data.” Edited by Timothee Poisot. <em>PLoS Computational Biology</em> 13 (2): e1005404. <a href="https://doi.org/10.1371/journal.pcbi.1005404">https://doi.org/10.1371/journal.pcbi.1005404</a>.</p>
</div>
<div id="ref-goss2009Ancient">
<p>Goss, E. M., I. Carbone, and N. J. Grünwald. 2009. “Ancient Isolation and Independent Evolution of the Three Clonal Lineages of the Exotic Sudden Oak Death Pathogen Phytophthora Ramorum.” <em>Molecular Ecology</em> 18 (6): 1161–74. <a href="https://doi.org/10.1111/j.1365-294X.2009.04089.x">https://doi.org/10.1111/j.1365-294X.2009.04089.x</a>.</p>
</div>
<div id="ref-Goss2009PLospathogens">
<p>Goss, E. M., M. Larsen, G. A. Chastagner, D. R. Givens, and N. J. Grünwald. 2009. “Population Genetic Analysis Infers Migration Pathways of Phytophthora Ramorum in Us Nurseries.” <em>PLoS Pathogens</em> 5 (9). <a href="https://doi.org/10.1371/journal.ppat.1000583">https://doi.org/10.1371/journal.ppat.1000583</a>.</p>
</div>
<div id="ref-Grunwald2012emergence">
<p>Grünwald, N. J., M. Garbelotto, E. M. Goss, K. Heungens, and S. Prospero. 2012. “Emergence of the Sudden Oak Death Pathogen Phytophthora Ramorum.” <em>Trends in Microbiology</em> 20 (3): 131–38. <a href="https://doi.org/10.1016/j.tim.2011.12.006">https://doi.org/10.1016/j.tim.2011.12.006</a>.</p>
</div>
<div id="ref-Grunwald2008ramorum">
<p>Grünwald, N. J., E. M. Goss, and C. M. Press. 2008. “Phytophthora Ramorum: A Pathogen with a Remarkably Wide Host Range Causing Sudden Oak Death on Oaks and Ramorum Blight on Woody Ornamentals.” <em>Molecular Plant Pathology</em> 9 (6): 729–40. <a href="https://doi.org/10.1111/j.1364-3703.2008.00500.x">https://doi.org/10.1111/j.1364-3703.2008.00500.x</a>.</p>
</div>
<div id="ref-Kamvar_2014">
<p>Kamvar, Zhian N., Javier F. Tabima, and Niklaus J. Grünwald. 2014. “Poppr: An R Package for Genetic Analysis of Populations with Clonal, Partially Clonal, and/or Sexual Reproduction.” <em>PeerJ</em> 2 (March): e281. <a href="https://doi.org/10.7717/peerj.281">https://doi.org/10.7717/peerj.281</a>.</p>
</div>
<div id="ref-Grunwald2015Novel">
<p>Kamvar, Z. N., J. C. Brooks, and N. J. Grünwald. 2015. “Novel R Tools for Analysis of Genome-Wide Population Genetic Data with Emphasis on Clonality.” <em>Frontiers in Genetics</em> 6 (JUN). <a href="https://doi.org/10.3389/fgene.2015.00208">https://doi.org/10.3389/fgene.2015.00208</a>.</p>
</div>
<div id="ref-Grunwald2016_vcfr">
<p>Knaus, B. J., and N. J. Grünwald. 2016. “Vcfr: A Package to Manipulate and Visualize Variant Call Format Data in R.” <em>Molecular Ecology Resources</em>. <a href="https://doi.org/10.1111/1755-0998.12549">https://doi.org/10.1111/1755-0998.12549</a>.</p>
</div>
<div id="ref-Knaus_2018inferring">
<p>Knaus, Brian J., and Niklaus J. Grünwald. 2018. “Inferring Variation in Copy Number Using High Throughput Sequencing Data in R.” <em>Frontiers in Genetics</em> 9 (April). <a href="https://doi.org/10.3389/fgene.2018.00123">https://doi.org/10.3389/fgene.2018.00123</a>.</p>
</div>
<div id="ref-tabima2018effectr">
<p>Tabima, Javier F, and Niklaus J Grunwald. 2018. “EffectR: An Expandable R Package to Predict Candidate Effectors.” <em>BioRxiv</em>, 398404. <a href="https://doi.org/10.1094/MPMI-10-18-0279-TA">https://doi.org/10.1094/MPMI-10-18-0279-TA</a>.</p>
</div>
</div>
</div>
