+++
# A Demo section created with the Blank widget.
# Any elements can be added in the body: https://sourcethemes.com/academic/docs/writing-markdown-latex/
# Add more sections by duplicating this file and customizing to your requirements.

active = true  # Activate this widget? true/false
weight = 35  # Order that this section will appear.

title = "Software, databases, tools and websites"
subtitle = ""

[design]
  # Choose how many columns the section has. Valid values: 1 or 2.
  columns = "1"

+++

## Nextflow  pipeline

:gear: [`nf-core/pathogensurveillance`](https://github.com/nf-core/pathogensurveillance): a population genomic pipeline for pathogen diagnosis, variant detection, and biosurveillance
[nf-core repository: [<i class="fab fa-github"></i>](https://github.com/nf-core/pathogensurveillance); development repository:
[<i class="fab fa-github"></i>](https://github.com/grunwaldlab/pathogensurveillance) 
]

## Python packages

:gear: `krisp`: A python package for designing diagnostic CRISPR-Cas Dx assays and other PCR-based diagnostic assays using whole genome or vcf data
[[preprint](https://www.biorxiv.org/content/10.1101/2023.11.16.567433v1) [<i class="fab fa-github"></i>](https://github.com/grunwaldlab/krisp)
]
![PyPI](https://img.shields.io/pypi/v/KRISP?label=pypi%20krisp)
![PyPI - Downloads](https://img.shields.io/pypi/dm/KRISP)

## R packages 

:gear: `poppr`: Analysis of population genetic data [[<i class="ai ai-open-access"></i>](https://peerj.com/articles/281/) [<i class="ai ai-open-access"></i>](https://doi.org/10.3389/fgene.2015.00208) [primer](https://grunwaldlab.github.io/Population_Genetics_in_R/) [<i class="fab fa-github"></i>](https://github.com/grunwaldlab/poppr) [news](https://grunwaldlab.github.io/poppr/)]

[![CRAN version](http://www.r-pkg.org/badges/version/poppr)](https://cran.r-project.org/package=poppr) 
[![Downloads from Rstudio mirror](https://cranlogs.r-pkg.org/badges/grand-total/poppr)](http://www.r-pkg.org/pkg/poppr) 
[![](http://cranlogs.r-pkg.org/badges/poppr)](https://cran.r-project.org/package=poppr)
[![Citation Badge](https://api.juleskreuer.eu/citation-badge.php?doi=10.7717/peerj.281)](https://peerj.com/articles/281/)

:gear: `vcfR`: QC and analysis of population genomic (vcf) data [[<i class="ai ai-open-access"></i>](https://doi.org/10.1111/1755-0998.12549) [<i class="ai ai-open-access"></i>](https://doi.org/10.3389/fgene.2018.00123) [primer](https://grunwaldlab.github.io/Population_Genetics_in_R/)
[<i class="fab fa-file-pdf-o"></i>](https://onlinelibrary.wiley.com/doi/abs/10.1111/1755-0998.12549) [primer](https://knausb.github.io/vcfR_documentation/vcfR_object.html) [<i class="fab fa-github"></i>](https://github.com/knausb/vcfR)]

[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/vcfR)](https://cran.r-project.org/package=vcfR)
[![](http://cranlogs.r-pkg.org/badges/grand-total/vcfR)](https://cran.r-project.org/package=vcfR)
[![](http://cranlogs.r-pkg.org/badges/vcfR)](https://cran.r-project.org/package=vcfR)
[![Citation Badge](https://api.juleskreuer.eu/citation-badge.php?doi=10.1111/1755-0998.12549)](https://doi.org/10.1111/1755-0998.12549)

:gear: `metacoder`: Create heat-trees and visualize microbiomes [[<i class="ai ai-open-access"></i>](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005404) [primer](https://grunwaldlab.github.io/metacoder_documentation/) [<i class="fab fa-github"></i>](https://github.com/grunwaldlab/metacoder)]

[![CRAN version](http://www.r-pkg.org/badges/version/metacoder)](https://cran.r-project.org/package=metacoder)
[![Downloads from Rstudio mirror](http://cranlogs.r-pkg.org/badges/grand-total/metacoder)](http://www.r-pkg.org/pkg/metacoder) 
[![Downloads from Rstudio mirror per month](http://cranlogs.r-pkg.org/badges/metacoder)](http://www.r-pkg.org/pkg/metacoder) 
[![Citation Badge](https://api.juleskreuer.eu/citation-badge.php?doi=10.1371/journal.pcbi.1005404)](https://doi.org/10.1371/journal.pcbi.1005404)

:gear: `taxa`: Manipulate hierarchical data [[<i class="ai ai-open-access"></i>](https://f1000research.com/articles/7-272/v1)  [<i class="fab fa-github"></i>](https://github.com/ropensci/taxa)]

[![cran version](http://www.r-pkg.org/badges/version/taxa)](https://cran.r-project.org/package=taxa)
[![](http://cranlogs.r-pkg.org/badges/grand-total/taxa)](https://cran.r-project.org/package=taxa)
[![rstudio mirror downloads](http://cranlogs.r-pkg.org/badges/taxa)](https://github.com/metacran/cranlogs.app)

:gear: `effectr`: Call effectors in pathogen genomes [[<i class="fab fa-github"></i>](https://github.com/grunwaldlab/effectR)]

[![cran version](http://www.r-pkg.org/badges/version/effectR)](https://cran.r-project.org/package=effectR)
[![](http://cranlogs.r-pkg.org/badges/grand-total/effectR)](https://cran.r-project.org/package=effectR)
[![rstudio mirror downloads](http://cranlogs.r-pkg.org/badges/effectR)](https://github.com/metacran/cranlogs.app)

:gear: `popprxl`: Import GenAlEx Excel data into `poppr` [[<i class="fab fa-github"></i>](https://github.com/grunwaldlab/popprxl)]

[![](http://cranlogs.r-pkg.org/badges/grand-total/popprxl)](https://cran.r-project.org/package=popprxl)
[![rstudio mirror downloads](http://cranlogs.r-pkg.org/badges/popprxl)](https://github.com/metacran/cranlogs.app)

## Educational resources related to R packages

- [Population genetics and genomics in R](http://grunwaldlab.github.io/Population_Genetics_in_R/)
- [Analysis of Microbiome Community Data in R](https://grunwaldlab.github.io/analysis_of_microbiome_community_data_in_r/)
- [Reproducible Science in R](http://grunwaldlab.github.io/Reproducible-science-in-R/)

## Websites

- [Phytophthora-ID](http://phytophthora-id.org/), a resource for the identification of *Phytophthora* species and genotypes using BLAST and R.
- [OomyceteDB](http://oomycetedb.cgrb.oregonstate.edu/): provides quality-controlled, aligned and annotated Oomycete _rps10_ sequences. _rps10_ is a mitochondrial locus chosen as a molecular barcode suitable for metabarcoding or amplicon sequencing specific to the Oomycetes. The curated database can be downloaded or queried using BLAST.
- [Microbe-ID](http://microbe-id.org/), a toolbox that can readily be customized for any taxonomic group or species for sequence based identification of species or marker-based identification of strains.
- [Github](https://github.com/grunwaldlab), the repository for resources developed by our lab.
- [OSF](https://osf.io/qg3sc/), repositories for data, code and related resources. 